src/main/java/uk/gov/companieshouse/chd/order/api/controller/MissingImageDeliveriesController.java

```diff
@@ -48,7 +56,10 @@ public MissingImageDeliveriesController(final CreateItemRequestValidator createM
             return ResponseEntity.status(BAD_REQUEST).body(new ApiError(BAD_REQUEST, errors));
         }
 
+        MissingImageDeliveriesRequest midRequest = mapper.mapMissingImageDeliveriesRequest(midDTO);
```

Comment:
Not sure about the name `MissingImageDeliveriesRequest` here. Typically wouldn't we use the term `request` to refer to what is passed in as a controller method parameter?

Reply:
The DTO that this maps to is actually a request object. Without the `Request` prefix the POJO appears ambiguous.

Reply:
As I say, it's more about code reader expectations when they read `...Request`. Not that important.